### PR TITLE
Fix #3 - Documentation

### LINKED ISSUE #3: Documentation
Add documentation to README.md

### PR DESCRIPTION
**Pull request by cyrilc-pro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
 ## Usage
-### Commands TBD
-* jfrog-support-bundle-flunky
-    - Arguments:
-        - case - The JFrog Support case number.
-    - Example:
+
+This plugin has a unique command `support-bundle` that:
+1. Creates a Support Bundle on the target Artifactory service
+2. Downloads the Support Bundle locally in a temporary file
```
Comment (ttetzlaff) [inline README.md:23]: "to a temporary file"

File: README.md [hunk 2]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
+3. Uploads the Support Bundle on JFrog "dropbox" service
```
Comment (ttetzlaff) [inline README.md:24]: or a to a specified Artifactory

File: README.md [hunk 3]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
+
+**Arguments**
+- `case` - The JFrog Support case number (required).
```
Comment (ttetzlaff) [inline README.md:27]: maybe `support-case` ?

File: README.md [hunk 4]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
+
+**Example**
 ```
-$ jfrog jfrog-support-bundle-flunky 1234
+$ jfrog support-bundle 1234
```
Comment (ttetzlaff) [inline README.md:31]: We should add an alias `sb` example too.

File: README.md [hunk 5]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
 ```
 
+**Optional flags**
+- `server-id` - The ID of the target Artifactory service in JFrog CLI configuration. Example: `--server-id=my-jfrog-service`
+- `download-timeout` - Timeout of the Support Bundle download. Example: `--download-timeout=10s`
+- `retry-interval` - Waiting time between a failed download attempt and the next attempt. Example: `--retry-interval=3s`
+- `prompt-options` - Ask what is to be included in the created Support Bundle. Example: `--prompt-options`
```
Comment (ttetzlaff) [inline README.md:38]: Specify what is to be...

File: README.md [hunk 6]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
...
-$ jfrog jfrog-support-bundle-flunky 1234
+$ jfrog support-bundle 1234
 ```
 
+**Optional flags**
+- `server-id` - The ID of the target Artifactory service in JFrog CLI configuration. Example: `--server-id=my-jfrog-service`
+- `download-timeout` - Timeout of the Support Bundle download. Example: `--download-timeout=10s`
+- `retry-interval` - Waiting time between a failed download attempt and the next attempt. Example: `--retry-interval=3s`
+- `prompt-options` - Ask what is to be included in the created Support Bundle. Example: `--prompt-options` 
+
```
Comment (ttetzlaff) [inline README.md:39]: Should we document `target-server-id`?

File: README.md [hunk 7]
```diff
@@ -17,15 +17,26 @@ Uninstalling the plugin
 `$ jfrog plugin uninstall jfrog-support-bundle-flunky`
 
 ## Usage
-### Commands TBD
-* jfrog-support-bundle-flunky
-    - Arguments:
-        - case - The JFrog Support case number.
-    - Example:
+
+This plugin has a unique command `support-bundle` that:
```
Comment (ttetzlaff) [inline README.md:21]: This plugin has a unique command `support-bundle` or `sb` that

File: README.md (hunk 4 above)
Comment (cyrilc-pro) [reply README.md:31]: We have it, I just don't know if it works...

File: README.md (hunk 5 above)
Comment (cyrilc-pro) [reply README.md:38]: Let's discuss this when we are satisfied with what we already planned

**Review by ttetzlaff — APPROVED** [review]

**Pull request merged**